src/lang.rs

```diff
@@ -1,295 +1,597 @@
+              "bitwise",
+              "contrib/RohanSreerama5/naiveBayesClassifier",
+              "contrib/anaisdg/anomalydetection",
+              "contrib/anaisdg/statsmodels",
+              "contrib/bonitoo-io/alerta",
+              "contrib/bonitoo-io/hex",
+              "contrib/bonitoo-io/servicenow",
+              "contrib/bonitoo-io/tickscript",
+              "contrib/bonitoo-io/victorops",
+              "contrib/bonitoo-io/zenoss",
+              "contrib/chobbs/discord",
+              "contrib/jsternberg/aggregate",
+              "contrib/jsternberg/influxdb",
+              "contrib/jsternberg/math",
+              "contrib/rhajek/bigpanda",
+              "contrib/sranka/opsgenie",
+              "contrib/sranka/sensu",
+              "contrib/sranka/teams",
+              "contrib/sranka/telegram",
+              "contrib/sranka/webexteams",
+              "contrib/tomhollingworth/events",
+              "csv",
+              "date",
+              "date/boundaries",
+              "dict",
+              "experimental",
+              "experimental/aggregate",
+              "experimental/array",
+              "experimental/bigtable",
+              "experimental/bitwise",
+              "experimental/csv",
+              "experimental/geo",
+              "experimental/http",
+              "experimental/http/requests",
+              "experimental/influxdb",
+              "experimental/iox",
+              "experimental/json",
+              "experimental/mqtt",
+              "experimental/oee",
+              "experimental/prometheus",
+              "experimental/query",
+              "experimental/record",
+              "experimental/table",
+              "experimental/universe",
+              "experimental/usage",
+              "generate",
+              "http",
+              "http/requests",
+              "influxdata/influxdb",
+              "influxdata/influxdb/monitor",
+              "influxdata/influxdb/sample",
+              "influxdata/influxdb/schema",
+              "influxdata/influxdb/secrets",
+              "influxdata/influxdb/tasks",
+              "influxdata/influxdb/v1",
+              "internal/boolean",
+              "internal/debug",
+              "internal/gen",
+              "internal/influxql",
+              "internal/location",
+              "internal/promql",
+              "internal/testutil",
+              "interpolate",
+              "join",
+              "json",
+              "kafka",
+              "math",
+              "pagerduty",
+              "planner",
+              "profiler",
+              "pushbullet",
+              "regexp",
+              "runtime",
+              "sampledata",
+              "slack",
+              "socket",
+              "sql",
+              "strings",
+              "system",
+              "testing",
+              "testing/expect",
+              "timezone",
+              "types",
+              "universe"
```

Comment:
Are these all of the stdlib packages or just some of them? My understanding is all of them. Then does it mean each time a new stdlib package is added, we should update this test list?

Reply:
Yeah, when a new version of flux brings in a new package, we'll have a couple of tests like this that will fail. The nice thing is that we're using a tool in our test machinations that allow us to do `UPDATE_EXPECT=true`, run the tests again, and they'll automatically update.